docs/manual/nullness-checker.tex

```diff
@@ -856,10 +856,16 @@ \section{Other tools for nullness checking\label{nullness-related-work}}
 
 If your code is already annotated with a different nullness
 annotation, the Checker Framework can type-check your code.
-It treats annotations from other tools
```

Comment:
The Interning, Lock, and Value checkers have the exact same wording as this.  Should we move the new text some and just refer it in those sections?